### ISSUE TITLE
Bug (spacing issue on line 1599) in latest commit (v4.0.6)

### ISSUE THREAD

**Issue by r4** [vmstan/gravity-sync#427]
Had some issues when deploying a 3 pihole solution that led to me digging through the code.
I found an minor bug in the latest commit for v4.0.6 on line 1599 of gravity-sync.

The space between "${LOCAL_FILE_OWNER}" and "${LOCAL_DNSMASQ_DIRECTORY}" is missing.

Line 1599: sudo chown ${LOCAL_FILE_OWNER}${LOCAL_DNSMASQ_DIRECTORY}/${PH_CNAME_CONF} >/dev/null 2>&1

You can see the diff here...
https://github.com/vmstan/gravity-sync/pull/424/files

I added the space back in and can confirm the bug goes away.

The output difference:
Before adding the space...
```
gravity-sync pull
∞ Initializing Gravity Sync (4.0.6)
✓ Loading gravity-sync.conf
✓ Detecting local Pi-hole installation
✓ Detecting remote Pi-hole installation
✓ Gravity Sync falling back to peerless mode
! Please configure Gravity Sync on remote host
✓ Evaluating arguments: PULL
» Remote target USER@IP
✓ Validating pathways to Pi-hole
✓ Validating pathways to DNSMASQ
✓ Hashing the remote Gravity Database
✓ Comparing to the local Gravity Database
! Differences detected in the Gravity Database
✓ Hashing the remote DNS Records
✓ Comparing to the local DNS Records
! Differences detected in the DNS Records
✓ Differences detected in the DNS Records
✓ Hashing the remote DNS CNAMEs
✓ Comparing to the local DNS CNAMEs
! Differences detected in the DNS CNAMEs
! Static DHCP Addresses not detected on the local Pi-hole
! Replication of Pi-hole settings is required
✓ Performing backup of local Gravity Database
✓ Performing backup of remote Gravity Database
✓ Checking Gravity Database copy integrity
✓ Pulling the remote Gravity Database
✓ Replacing the local Gravity Database
✓ Setting file ownership on Gravity Database
✓ Setting file permissions on Gravity Database
✓ Performing backup of local DNS Records
✓ Performing backup of remote DNS Records
✓ Pulling the remote DNS Records
✓ Replacing the local DNS Records
✓ Setting file ownership on DNS Records
✓ Setting file permissions on DNS Records
✓ Performing backup of local DNS CNAMEs
✓ Performing backup of remote DNS CNAMEs
✓ Pulling the remote DNS CNAMEs
✓ Replacing the local DNS CNAMEs
✗ Setting file ownership on DNS CNAMEs
```

After adding the space...
```
gravity-sync pull
∞ Initializing Gravity Sync (4.0.6)
✓ Loading gravity-sync.conf
✓ Detecting local Pi-hole installation
✓ Detecting remote Pi-hole installation
✓ Gravity Sync falling back to peerless mode
! Please configure Gravity Sync on remote host
✓ Evaluating arguments: PULL
» Remote target USER@IP
✓ Validating pathways to Pi-hole
✓ Validating pathways to DNSMASQ
✓ Hashing the remote Gravity Database
✓ Comparing to the local Gravity Database
! Differences detected in the Gravity Database
✓ Hashing the remote DNS Records
✓ Comparing to the local DNS Records
! Differences detected in the DNS Records
✓ Differences detected in the DNS Records
✓ Hashing the remote DNS CNAMEs
✓ Comparing to the local DNS CNAMEs
! Differences detected in the DNS CNAMEs
! Static DHCP Addresses not detected on the local Pi-hole
! Replication of Pi-hole settings is required
✓ Performing backup of local Gravity Database
✓ Performing backup of remote Gravity Database
✓ Checking Gravity Database copy integrity
✓ Pulling the remote Gravity Database
✓ Replacing the local Gravity Database
✓ Setting file ownership on Gravity Database
✓ Setting file permissions on Gravity Database
✓ Performing backup of local DNS Records
✓ Performing backup of remote DNS Records
✓ Pulling the remote DNS Records
✓ Replacing the local DNS Records
✓ Setting file ownership on DNS Records
✓ Setting file permissions on DNS Records
✓ Performing backup of local DNS CNAMEs
✓ Performing backup of remote DNS CNAMEs
✓ Pulling the remote DNS CNAMEs
✓ Replacing the local DNS CNAMEs
✓ Setting file ownership on DNS CNAMEs
✓ Setting file permissions on DNS CNAMEs
✓ Updating local FTLDNS configuration
✓ Reloading local FTLDNS services
› Performing replicator diagnostics
✓ Rehashing the remote Gravity Database
✓ Recomparing to local Gravity Database
✓ Rehashing the remote DNS Records
✓ Recomparing to local DNS Records
✓ Recomparing to local DNS Records
✓ Rehashing the remote DNS CNAMEs
✓ Recomparing to local DNS CNAMEs
! Static DHCP Addresses not detected on the local Pi-hole
✓ Logging successful PULLhes∞ Logging successful PULL
∞ Gravity Sync PULL completed after 13 seconds

```

**Comment by pviglucci** [comment]
Thanks for this. Fix worked for me as well.

**Comment by vmstan** [comment]
Fixed by #431 which is now published as v4.0.7
